apps/bfd-model/bfd-model-rif/src/main/java/gov/cms/bfd/model/rif/parse/RifParsingUtils.java

```diff
@@ -127,6 +144,19 @@ public static Integer parseInteger(String intText) {
     }
   }
 
+  /**
+   * @param longText the number string to parse
+   * @return an {@link Optional} populated with an {@link Long} if the input has data, or an empty
+   *     Optional if not
+   */
```

Comment:
⚠️ Docs missing descriptions, which may be enforced in the future.

Reply:
They are missing everywhere in this file but I'll clean that up.